control_panel_api/aws.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+import boto3
+
+
+def create_bucket(name, **kwargs):
+    boto3.client('s3').create_bucket(
+        Bucket=name,
+        ACL=kwargs.get('acl', ''),
```

Comment:
I would default to `private` here for the `acl` argument.

Reply:
👍